### PR TITLE
initial commit for dtapi testing (common pom and first classes)

### PR DESCRIPTION
**Pull request by promahos**

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/pageobjects/DashboardPage.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.softserveinc.ita.pageobjects;
+
+public class DashboardPage extends MainMenu {
```
Comment (Nikodych) [inline src/main/java/com/softserveinc/ita/pageobjects/DashboardPage.java:3]: as I understand, MainMenu as other pageobjects are admin pages, and probably need to create a separate admin package in pageobjects package

File: src/test/java/com/softserveinc/ita/ProtocolTest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import com.softserveinc.ita.pageobjects.ProtocolPage;
+import com.softserveinc.ita.pageobjects.util.TestRunner;
+import org.testng.annotations.Test;
+
+public class ProtocolTest extends TestRunner {
+
+    @Test
+    public void verifyProtocolPageOpening() {
+        // TODO: 13.05.2022 use config files for credentials and expected url
+        ProtocolPage protocolPage = new LoginPage().login("admin", "dtapi_admin").openProtocolPage();
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/ProtocolTest.java:17]: new LoginPage()
.login("admin", "dtapi_admin")
.openProtocolPage();

File: src/test/java/com/softserveinc/ita/ProtocolTest.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        String expectedUrl = "https://dtapi.if.ua/admin/protocol";
```
Comment (Nikodych) [inline src/test/java/com/softserveinc/ita/ProtocolTest.java:18]: var

**Review by Nikodych — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/pageobjects/LoginPage.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import static org.openqa.selenium.By.name;
+import static org.openqa.selenium.By.xpath;
+
+public class LoginPage {
+
+    public DashboardPage login(String login, String password) {
+
+        $(name("username")).sendKeys(login);
+        $(name("password")).sendKeys(password);
+        $(xpath("//button[@class='butt-wrap mat-flat-button mat-button-base mat-primary']")).click();
```
[the remark this answers is not in the document]
Comment (promahos) [reply src/main/java/com/softserveinc/ita/pageobjects/LoginPage.java]: yes, that's better, thank you

**Pull request merged**